test/train-sets/ref/cb_adf_crash2.stderr

```diff
@@ -8,11 +8,11 @@ Reading datafile = train-sets/cb_adf_crash_2.data
 num sources = 1
 average  since         example        example  current  current  current
 loss     last          counter         weight    label  predict features
-    n.a.     n.a.            1            1.0  unknown        0:1...        3
+    n.a.     n.a.            1            1.0  unknown        0:1...        2
 
 finished run
 number of examples = 1
 weighted example sum = 1.000000
 weighted label sum = 0.000000
 average loss = n.a.
-total feature number = 3
+total feature number = 2
```

Comment:
why is the number of features changing? I assume you removed an empty line?

Reply:
As above.

Reply:
Yes.  This is coming from the empty line.